go/pkg/filemetadata/cache_posix_test.go

```diff
@@ -0,0 +1,34 @@
+// +build !windows
```

Comment:
Out of curiosity - what does this line do?

Reply:
Ah I think https://golang.org/pkg/go/build/#hdr-Build_Constraints. I wonder if its necessary given the selects, but lets leave it for now since you're probably reapplying the patch from Fumitoshi.

Reply:
I think it tells go build to not build this file if the platform is windows. For bazel, the same is achieved via the BUILD file.